doc/README.md

```diff
@@ -191,33 +194,34 @@ paths with the highest cum weight.
   browser to view it.
 * **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
 
-### Interpreting the Callgraph
+### Interpreting the Nodes
+
+![callgraph1](images/callgraph_nodes.svg)
```

Comment:
The picture has a typo: "defalte" instead of "deflate". I viewed the rendered version at https://github.com/wyk9787/pprof/blob/readme/doc/README.md.

I would also remove all callouts from the example images. Currently the callouts look like they are part of the generated callgraph itself, it's hard to tell that these are explanatory callouts. It's better to explain various elements in the documentation text referring to nodes in the example call graph by name. This will also make it easier to review and update this doc.

Reply:
Done.